sql/sys_vars.cc

```diff
@@ -5352,11 +5352,12 @@ static Sys_var_charptr Sys_wsrep_node_incoming_address(
        PREALLOCATED GLOBAL_VAR(wsrep_node_incoming_address),CMD_LINE(REQUIRED_ARG),
        IN_SYSTEM_CHARSET, DEFAULT(WSREP_NODE_INCOMING_AUTO));
 
+static PolyLock_mutex PLock_wsrep_slave_threads(&LOCK_wsrep_slave_threads);
```

Comment:
`PLock_wsrep_slave_threads` is not used anywhere in this file, should it be removed or should the code below use it?